src/environment.h

```diff
@@ -379,6 +378,11 @@ class ServerEnvironment : public Environment
 	// is weather active in this environment?
 	bool m_use_weather;
 	ABMHandler * m_abmhandler;
```

Comment:
must be initialized after m_script->initializeEnvironment(m_env); add_legacy_abms(m_env, m_nodedef); 
maybe better to do via init() func